app/controllers/cities_controller.rb

```diff
@@ -6,15 +6,15 @@ def index
     @cities = City.all
     # render json: @cities.map { |city| CitySerializer.new(city).serializable_hash[:data][:attributes] }
```

Comment:
- [ ] Kindly, remove this old commented code so your repo's cleanliness is improved.